README.md

```diff
@@ -1 +1,67 @@
-Tools for Wizzilab products
+# AppLink via python
+
+Python scripts to communicate with Wizzilab's AppLink. 
+
+- watch_reports.py allows to display reports coming from sensors
+- read_example.py allows to read a property of a device 
+- write_example.py shows how to configure a property of a device. 
+
+## Installation and example
```

Comment:
This section should be split into an "Installation" section and a "Tools" section.
Because we decided those are tools, and not examples anymore.
Also, the Tools section should be split into a subsection per tool.